frontend_setup.sh

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
+
```

Comment:
You can try using options in your script. This help change the behaviour of your scripts. For example, you can add this ```set -o verbose``` right below your shebang.